rdr_service/resource/calculators/participant_ubr.py

```diff
@@ -48,8 +62,10 @@ def ubr_gender_identity(birth_sex, gender_ident, gender_ident_closer):
                         survey.
         :param gender_ident_closer: Comma delimited str of answer codes to GenderIdentity_SexualityCloserDescription
                         question in "TheBasics" survey.
-        :return: 1 if UBR else 0
+        :return: UBRValueEnum
         """
+        if gender_ident is None and gender_ident_closer is None:
```

Comment:
Just confirming that the PMI_Skip answer code is not a factor for these (along with checking for None)?

Reply:
The Skip value get tested in the code later when the two value sets are combined.